cmd/plugin.go

```diff
@@ -105,8 +105,21 @@ var (
 
 			log.Info().Str("module", "plugin").Msgf("Start plugin %s %s", pluginName, exeArgs)
 
+			logfile := viper.GetString("log")
+			if logfile == "" {
+				logfile = fmt.Sprintf("%s/%s.log", pluginDir, pluginName)
+			}
+
+			f, err := os.OpenFile(logfile, os.O_RDWR|os.O_CREATE, 0644)
```

Comment:
how about add append flag for the logfile?
logfile is overwritten, if exist.
```
diff --git a/cmd/plugin.go b/cmd/plugin.go
index 9944cdb..6b5f4a3 100644
--- a/cmd/plugin.go
+++ b/cmd/plugin.go
@@ -110,7 +110,7 @@ var (
                                logfile = fmt.Sprintf("%s/%s.log", pluginDir, pluginName)
                        }

-                       f, err := os.OpenFile(logfile, os.O_RDWR|os.O_CREATE, 0644)
+                       f, err := os.OpenFile(logfile, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0644)
                        if err != nil {
                                log.Info().Str("module", "plugin").Err(err)
                                return err
```

Reply:
@heejin-github Good idea. I gonna add it. thx.

Reply:
@heejin-github Added at `20d1cc1`